modules/default/weather/weatherprovider.js

```diff
@@ -126,19 +158,31 @@ const WeatherProvider = Class.extend({
 	 * @param {string} type what contenttype to expect in the response, can be "json" or "xml"
 	 * @returns {Promise} resolved when the fetch is done
 	 */
-	fetchData: async function (url, type = "json") {
-		url = this.getCorsUrl() + url;
+	fetchData: async function (url, method = "GET", type = "json", requestHeaders = undefined, expectedResponseHeaders = undefined) {
```

Comment:
fetchData is used in all providers so changing the params order here will break at least the `envcanada.js` provider and AFAIS `method` is not used inside the `fetchData` function.

Reply:
Ah, you're right, nicely spotted. When merging my changes into the development-branch I didn't catch that the method-parameter was removed in https://github.com/MichMich/MagicMirror/pull/2935. I'll fix it 😊